doc/classes/Basis.xml

```diff
@@ -219,18 +219,18 @@
 				[gdscript]
 				# Rotate this Node3D every frame.
 				func _process(delta):
-				    basis = basis.rotated(Vector3.UP, TAU * delta)
-				    basis = basis.rotated(Vector3.RIGHT, TAU * delta)
+					basis = basis.rotated(Vector3.UP, TAU * delta)
+					basis = basis.rotated(Vector3.RIGHT, TAU * delta)
 
-				    basis = basis.orthonormalized()
+					basis = basis.orthonormalized()
 				[/gdscript]
 				[csharp]
 				// Rotate this Node3D every frame.
 				public override void _Process(double delta)
 				{
-				    Basis = Basis.Rotated(Vector3.Up, Mathf.Tau * (float)delta)
-				                 .Rotated(Vector3.Right, Mathf.Tau * (float)delta)
-				                 .Orthonormalized();
+					Basis = Basis.Rotated(Vector3.Up, Mathf.Tau * (float)delta)
+								 .Rotated(Vector3.Right, Mathf.Tau * (float)delta)
+								 .Orthonormalized();
```

Comment:
Should probably check the whole diff for other occurrences of `\t `.
```suggestion
								.Rotated(Vector3.Right, Mathf.Tau * (float)delta)
								.Orthonormalized();
```

Reply:
Some more I found:

`@GlobalScope.xml` (legit)
![image](https://github.com/godotengine/godot/assets/2223172/7dda8b26-518a-4447-ad03-3b3a9eb82bb7)
`Control.xml` (fixed)
`DisplayServer.xml` (legit)
![image](https://github.com/godotengine/godot/assets/2223172/fec3d220-816d-4522-9b37-bc61d99b590d)
`Node.xml` (legit)
![image](https://github.com/godotengine/godot/assets/2223172/9e883771-9a80-4766-8a58-56a5f2d09078)
`RenderingDevice.xml`  (fixed)
`RenderingServer.xml` (legit?)
![image](https://github.com/godotengine/godot/assets/2223172/18881dcf-042c-41a9-ba91-648dae5c1927)
`SkeletonProfileHuman.xml` (legit)
![image](https://github.com/godotengine/godot/assets/2223172/fec62b01-d34d-4677-b5f3-5df141af8fbd)